src/main/java/net/wasdev/wlp/ant/install/WasDevInstaller.java

```diff
@@ -136,10 +136,9 @@ public void install(InstallLibertyTask task) throws Exception {
     }
 
     private String getRuntimeURI(LibertyInfo selected) {
-        String propertyName = ("webProfile6".equalsIgnoreCase(type)) ? "uri" : type;
-        String value = selected.getProperty(propertyName);
+        String value = selected.getProperty(type);
```

Comment:
I'm still wondering about if they are using an older Liberty runtime, then wouldn't webProfile6 still be valid? In which case maybe this logic needs to stay?

Reply:
I addressed this in an edit to my previous comment.